Library/Koala/Koala.swift

```diff
@@ -1769,14 +1774,14 @@ private func properties(category category: KsApi.Category) -> [String:AnyObject]
     .withAllValuesFrom(parentProperties.prefixedKeys("parent_"))
 }
 
-private func properties(shareContext shareContext: ShareContext,
-                                     loggedInUser: User?,
-                                     shareActivityType: String? = nil) -> [String:AnyObject] {
+private func properties(shareContext: ShareContext,
+                        loggedInUser: User?,
+                        shareActivityType: UIActivityType? = nil) -> [String:Any] {
 
-  var result: [String:AnyObject] = [:]
+  var result: [String:Any] = [:]
 
-  result["share_activity_type"] = shareActivityType
-  result["share_type"] = shareTypeProperty(shareActivityType)
+  result["share_activity_type"] = shareActivityType?.rawValue
+  result["share_type"] = shareActivityType.flatMap(shareTypeProperty)
```

Comment:
I was concerned because the type of the Dictionary is `[String:Any]` not `[String:Any?]`, but I see it just won't exist if you set it to nil?